src/lib/plugins/plugin.module.ts

```diff
@@ -52,7 +55,8 @@ Model.register('columnChooser', ColumnChooserModel)
 		RestModule,
 		StatusBarModule,
 		TabTrapModule,
-		TitleModule
+		TitleModule,
+		ReactiveFormsModule
```

Comment:
is it required by autocomplete? can you post a link

Reply:
its required by reactive forms, we could use template-driven forms instead, but with reactive forms it would be much easier to subscribe to value changes (as its said in docs https://material.angular.io/components/autocomplete/overview)